packages/design-system/src/components/Badge/style.scss

```diff
@@ -0,0 +1,30 @@
+@import '../../scss/variables/colors';
+
+.badge {
+  display: inline-block;
+  box-sizing: border-box;
+  padding: 0 3px;
+  height: 18px;
+  min-width: 18px;
+
+  font-size: 10px;
+  font-weight: 600;
+  line-height: 18px;
+  text-align: center;
+  border-radius: 18px;
+  overflow: hidden;
+
+  & + & {
+    margin-left: 4px;
+  }
+}
+
+.badge--primary {
```

Comment:
I think this class is not necessary, those colors could be used in `.badge` class